server/build.gradle

```diff
@@ -151,3 +154,29 @@ task buildDocker(type: Docker) {
         }
     }
 }
+
+task buildStandalone() {
```

Comment:
For building the extender server for standalone operation without requiring Docker